manim/mobject/probability.py

```diff
@@ -229,106 +234,216 @@ def construct(self):
                 ]
                 colors = ["#003f5c", "#58508d", "#bc5090", "#ff6361", "#ffa600"]
                 bar = BarChart(
-                    pull_req,
-                    max_value=max(pull_req),
-                    bar_colors=colors,
-                    bar_names=versions,
-                    bar_label_scale_val=0.3,
+                    values = pull_req,
+                    bar_names = versions,
+                    bar_colors = colors
                 )
                 self.add(bar)
     """
 
     def __init__(
         self,
         values: Iterable[float],
-        height: float = 4,
-        width: float = 6,
-        n_ticks: int = 4,
-        tick_width: float = 0.2,
-        label_y_axis: bool = True,
-        y_axis_label_height: float = 0.25,
-        max_value: float = 1,
-        bar_colors=[BLUE, YELLOW],
-        bar_fill_opacity: float = 0.8,
-        bar_stroke_width: float = 3,
-        bar_names: List[str] = [],
-        bar_label_scale_val: float = 0.75,
-        **kwargs
-    ):  # What's the return type?
-        VGroup.__init__(self, **kwargs)
-        self.n_ticks = n_ticks
-        self.tick_width = tick_width
-        self.label_y_axis = label_y_axis
-        self.y_axis_label_height = y_axis_label_height
-        self.max_value = max_value
+        bar_names: Optional[Iterable[str]] = None,
+        x_length: Optional[float] = None,
+        x_label_constructor=Text,
+        x_label_buff=None,
+        x_label_scale_value: Optional[float] = None,
+        y_length: Optional[float] = config["frame_height"] - 4,
+        y_range: Optional[Sequence[float]] = None,
+        y_step: Optional[float] = None,
+        y_include_numbers: Optional[bool] = True,
+        y_number_scale_value: Optional[float] = 0.75,
```

Comment:
I think it might be easier to understand if the most important parameters came first.
```suggestion
        y_range: Optional[Sequence[float]] = None,
        y_step: Optional[float] = None,      
        x_length: Optional[float] = None,
        y_length: Optional[float] = config["frame_height"] - 4,  
        x_label_constructor=Text,
        x_label_buff=None,
        x_label_scale_value: Optional[float] = None,
        y_include_numbers: Optional[bool] = True,
        y_number_scale_value: Optional[float] = 0.75,
```